### PR TITLE
chore(docs): add auth info to the rest api docs

### PR DESCRIPTION
**Pull request by carlosmunoz**
Also add some info to the README on how to build the enunciate documentation.

---
This change is [Reviewable](https://reviewable.io/reviews/zanata/zanata-api/49)

### CODE REVIEW

**Comment by seanf** [comment]
@carlosmunoz regarding the enforcer failure, you should use this: https://github.com/zanata/zanata-client/pull/112/commits/101f74de2915a2fc7c6077adb246e9086da66e3d

File: README.txt [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+
+### REST API Documentation
+
+Documentation for this API can be generated by calling
+
+`mvn enunciate:docs`
+
+_(Please note this has to run with JDK 1.7 at the moment)_
+
+Once generated, you can find the documentation at `target/enunciate/build/docs/rest-api-docs/index.html`
```
Comment (seanf) [inline README.txt]: What directory is that relative to?

**Comment by aeng** [comment]
Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-api/49)*

**Comment by djansen-redhat** [comment]
:white_check_mark: Looks fine

**Pull request merged**